### PR TITLE
Feature add new reservation

### PR DESCRIPTION
**Pull request by fezzopro**
## PR Details 

This Pull request contains: 

- [ ] #10 
- [ ] #7 
- [ ] #4 
- [ ] #1

### CODE REVIEW

File: src/modules/components.js [hunk 1]
```diff
@@ -61,19 +65,107 @@ class Components {
...
+      document.querySelector('.reservations-div').remove();
+    }
+
+    const ul = document.createElement('ul');
+    reservationArray.reverse().forEach((reservation) => {
+      const li = document.createElement('li');
+      li.textContent = `${reservation.date_start} - ${reservation.date_end} by ${reservation.username}`;
+      ul.appendChild(li);
+    });
+    div.appendChild(ul);
```
Comment (sunga12) [inline src/modules/components.js:147]: In the Browser, this doesnt quite show as intended, as it shows only the start date for both dates as seen in the screenshot below

**Review by sunga12 — CHANGES_REQUESTED** [review]
Hey @fezzopro  , 



Great work So far on the reservations! Just a few suggested changes to make before we can merge this.

## Highlights
Correct use of GitHub Flow ✔
All linter checks are passing ✔
Descriptive Commits✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Reducing the size of your Images on the Reservation Popup, so the user doesn't need to scroll down so much to see the entire photo

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/components.js (hunk 1 above)
Comment (fezzopro) [reply src/modules/components.js]: Thanks for spotting this. I have fixed the issue and pushed the changes.
Thanks for the feedback 👍

**Review by sunga12 — APPROVED** [review]
# STATUS APPROVED 🟢 

Hey @fezzopro 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

onward-quest



## Highlights
 - The Dates are correct now💯 

## Optional suggestions

- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------

**Pull request merged**